app/views/OfferedCourses/CourseItem.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Text, View, Image, StyleSheet, ScrollView} from 'react-native';
+import {Button} from "native-base";
+import globalStyles from "../../styles";
+
+class CourseItem extends Component {
+
+    static propTypes = {
+        course: PropTypes.object.isRequired,
+    };
+
+    render() {
+
+        const { course } = this.props;
+
+        return (
+            <View style={styles.course}>
+                <Image
+                    source={{uri: 'https://facebook.github.io/react/logo-og.png'}}
+                    style={styles.image}
+                />
+                <View style={styles.info}>
+                    <Text style={styles.title}>{course.name.en}</Text>
+                    <Text>{course.description.en}</Text>
+                <View>
+                    <Button bordered rounded dark style={globalStyles.editButton}
+                        title="Learn More">
+                        <Text> Learn More </Text>
+                    </Button>
+                </View>
+                </View>
+                </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Move these styles also to the styles file, and import that here. We should also come up with naming scheme for styles.

Reply:
This styles made temporarily, no need to move anything yet